HackIllinois/UI/TableView/Cells/HIBubbleCell/HIEventCell.swift

```diff
@@ -38,20 +40,41 @@ class HIEventCell: HIBubbleCell {
 
```

Comment:
Make sure to clean up the code in HIEventCell by removing whitespace and commented out code.

Also, please let the color of the Event Type text to \.baseText and make sure the text isn't fully uppercase. You can use the code:

`        eventTypeLabel.text = eventType.description.lowercased().capitalized`